### DISCUSSION TITLE
Cron to update Website from master on 5Apps

### DISCUSSION THREAD

**Discussion by JustinBack** [tosdr/tosdr.org#255] [Ideas] [answered]
Do we have a cron yet to update the website from master on 5Apps? The latest revision of the site is almost 30 days old

**Comment by michielbdejong** [comment, marked as answer]
Good point, I'll set this up! We can use our server at SecureDragon for this (the one that hosts https://tosback.org)

**Comment by michielbdejong** [comment]
The script is https://github.com/tosdr/edit.tosdr.org/blob/master/db/deploy.sh
I'm running it manually now
